MACOS.md

```diff
@@ -198,14 +184,67 @@ brew upgrade openssl     || brew install openssl
+```
+
+Now, add VS Code to your Dock by right-clicking on the icon to bring up the context menu and choosing Options, Keep in Dock.
+
+### Launching from the terminal
+
+In VS Code, open the Command Palette (`CMD` + `SHIFT` + `P`), type `shell command` and click on "Shell Command: Install 'code' command in PATH" :point_down:
+
+![](images/mac_vscode_command.png)
+
+Now quit the VS Code, quit the Terminal (`CMD` + `Q`) and restart it.
+
+Try typing `code` :point_right: if VS Code opens in new window, you can proceed to the next point!
+
+
+## VS Code Extensions
+
+### Installation
+
+Let's install some useful extensions to VS Code. Copy the command below then paste it in your terminal:
+
+```bash
+code --install-extension ms-vscode.sublime-keybindings MS-vsliveshare.vsliveshare misogi.ruby-rubocop hridoy.rails-snippets dbaeumer.vscode-eslint
+```
+
+Here is a list of the extensions you are installing:
+- [Sublime Text Keymap and Settings Importer](https://marketplace.visualstudio.com/items?itemName=ms-vscode.sublime-keybindings)
+- [Live Share](https://marketplace.visualstudio.com/items?itemName=MS-vsliveshare.vsliveshare)
+- [ruby-rubocop](https://marketplace.visualstudio.com/items?itemName=misogi.ruby-rubocop)
+- [Ruby on Rails](https://marketplace.visualstudio.com/items?itemName=hridoy.rails-snippets)
+- [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint)
+
+### Live Share configuration
+
+[Visual Studio Live Share](https://visualstudio.microsoft.com/services/live-share/) is a VS Code extension which allows you to share the code in your text editor for debugging and pair-programming: let's set it up!
 
-A text editor is one of the most important tools of a developer. Go to [this page](http://www.sublimetext.com/3) and download **Sublime Text 3** for OS X. Install it (double click the downloaded file and drag & drop the app **into** the `Applications` folder, **don't skip this**). If you had Sublime Text 2 installed before, please uninstall it (by dragging/dropping it to the Trash).
+Click on the little arrow at the bottom of the left bar :point_down:
 
-Sublime Text is free without any time limitation but a popup will appear every ten saves to remind you there is a license to buy. You can hit `Esc` when this happens, but feel free to buy Sublime Text if you really like this one (there are alternatives).
+![](images/liveshare.png)
 
-Again, make sure that Sublime Text is there, not in the disk image you downloaded. To make sure it's correct, once Sublime Text is installed, unmount the "Sublime Text 3" disk in the left panel of Finder. Finder will complain if something went wrong. Ask a teacher.
+- Click on the "Share" button, then on "GitHub (Sign in using GitHub account)".
+- A popup appears asking you to sign in with GitHub: click on "Allow".
+- You are redirected to a GitHub page in you browser asking you to authorize Visual Studio Code: click on "Continue" then "Authorize github".
```

Comment:
Let's move this part after the Github section to make sure they all have their account setup 👌

Reply:
Actually, it's already after the GitHub section for all three setups :ok_hand: